src/map/status.cpp

```diff
@@ -5627,6 +5628,10 @@ void status_calc_state( block_list& bl, status_change& sc, std::shared_ptr<s_sta
 					}
 					break;
 
+				case SC_LONGING:
```

Comment:
One thing I wonder is... what's the point of having configurable flags when we need to add all SCs here anyway that have these flags defined?

Also, shouldn't this then be for pre-re only?

Reply:
This is only necessary to define the conditional ones. Not all of them.